package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "spaniel",
-  "version": "2.5.6",
+  "version": "2.5.7",
```

Comment:
why are we bumping the version within this PR? ideally we create a separate PR for the patch release.